daemon/server.go

```diff
@@ -4,17 +4,26 @@ import (
 	"context"
 	"fmt"
 	"net/http"
+	"os"
 
 	"github.com/docker/docker/api/types"
 	docker "github.com/docker/docker/client"
 
 	"github.com/ubclaunchpad/inertia/common"
 )
 
+// daemonVersion indicates the daemon's corresponding Inertia daemonVersion
+var daemonVersion string
+
 const (
 	// specify location of deployed project
 	projectDirectory = "/app/host/project"
 
+	// specify location of SSL certificate
+	sslDirectory  = "/app/host/ssl/"
+	daemonSSLCert = sslDirectory + "daemon.cert"
+	daemonSSLKey  = sslDirectory + "daemon.key"
```

Comment:
Might as well use Join

Reply:
Don't think we can if we want to keep these as `const` 🤔